lib/rules/web/admin_console/4.5/clustersettings.xyaml

```diff
@@ -0,0 +1,26 @@
+click_update_channel_button:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='current-channel-update-link']
+    op: click
+click_channels_drop_down:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-button']
+    op: click
+check_drop_down_items:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-menu' and contains(., '<menu_item_1>')]
+  - selector:
+      xpath: //button[@data-test-id='dropdown-menu' and contains(., '<menu_item_2>')]
+  - selector:
+      xpath: //button[@data-test-id='dropdown-menu' and contains(., '<menu_item_3>')]
+check_list_of_channels_in_channel_modal:
+  params:
+    menu_item_1: stable-<product_version>
+    menu_item_2: fast-<product_version>
+    menu_item_3: candidate-<product_version>
+  action: click_update_channel_button
+  action: click_channels_drop_down
```

Comment:
as [above comment](https://github.com/openshift/verification-tests/pull/1522/files#r487615194), here we can use `action: click_dropdown_button` instead.

Reply:
@yapei update:
- removed the redundant rule and used the existing web rule as suggested.